### PR TITLE
fix: solve not correct value displayed when max btn is used

### PR DESCRIPTION
**Pull request by lucasgabrielgsp**
- [x] solve not correct value displayed when max btn is used

### CODE REVIEW

File: source/pages/Send/SendSys.tsx [hunk 1]
```diff
@@ -60,8 +60,18 @@ export const SendSys = () => {
     ? Object.values(activeAccount.assets.syscoin)
     : [];
 
+  const formattedAssetBalance =
+    selectedAsset &&
+    truncate(
+      formatCurrency(
+        String(+selectedAsset.balance / 10 ** selectedAsset.decimals),
```
Comment (claudiovb) [inline source/pages/Send/SendSys.tsx:67]: The fix seems to work well, but I noticed, for now, we have any type of syscoin assets. It's best to ensure we always will have decimals field for syscoin assets.
https://github.com/syscoin/pali-wallet/blob/ba7eb368b2a102793e1f86acc1768a3ec7f808be/source/state/vault/types.ts#L49

**Review by claudiovb — CHANGES_REQUESTED** [review]
Type syscoin assets and make tests through blockbook calls

**Comment by leozera14** [comment]
Code seems legit and looks that fix the issue, only need to fix my Pali build to test it first before I can say it can be merged.

**Comment by leozera14** [comment]
I test it, found an error about decimals coming as undefined but already fixed it. Now is working and can merge it.

**Review by claudiovb — DISMISSED** [review]

**Review by claudiovb — APPROVED** [review]

**Pull request merged**